crt_portal/cts_forms/templates/forms/pro_template.html

```diff
@@ -0,0 +1,99 @@
+{% extends "forms/report_base.html" %}
+
+{{ super }}
+{% block form_id %}
+<form id="report-form"
+      class="usa-form"
+      action=""
+      method="post"
+      novalidate
+>
+{% endblock form_id %}
+
+{% block form_questions %}
+<a name="Intake"></a>
+<div class="crt-portal-card">
+  <div class="crt-portal-card__content crt-portal-card__content--lg">
+        {% if page_errors %}
+          <section id="page-errors" class="display-none page-errors margin-bottom-2" role="alert" aria-live="assertive">
+            {{ num_page_errors }} error{{ num_page_errors|pluralize }}: {{ page_errors_desc }}
+          </section>
+        {% endif %}
+
+        <a name="Intake"></a>
```

Comment:
We could replace these two duplicated and empty anchor tags w/ an `id="Intake"` attribute on the element most relevant for each section. The `h2` could be appropriate here.